### DISCUSSION TITLE
Method Chaining?

### DISCUSSION THREAD

**Discussion by 0-don** [royNiladri/js-big-decimal#96] [Ideas]
Wouldnt it be a good feature to enable method chaining by returing itself with `this` to make something like this posible

```ts
bigDecimal.multiple(10,10).divide(10).round(2)
```

**Comment by 0-don** [comment]
I see its already posible by defining an instance

**Comment by 0-don** [comment]
but it probably be cool if with the static methods it would return the result as an instance

**Reply by royNiladri** [reply]
Seems like a good idea! Will pick it up once I have bandwidth.
